schemas/data/src/locales/en-CA.ftl

```diff
@@ -82,7 +82,7 @@ error-ActivityType-ResultAnalyticalMethod = Requires LaboratoryName and ResultAn
 error-CharacteristicName-MethodSpeciation = MethodSpeciation required for selected CharacteristicName
 error-CharacteristicName-Nutrient-ResultSampleFraction = ResultSampleFractions for nutrients should follow guidance in Best Practices Guide for Nutrient Data (see resources page or contact us)
 error-CharacteristicName-ResultSampleFraction = ResultSampleFraction required for selected CharacteristicName
-error-CharacteristicName-StableIsotope-MethodSpeciation = Isotope MethodSpeciation required for selected CharacteristicName
+error-CharacteristicName-StableIsotope-MethodSpeciation = Isotope MethodSpeciation required for selected CharacteristicName. For guidance, please see Best Practices for Reporting Isotope Data on the Resources page
```

Comment:
Something to keep in mind for the future, these can link to the pages.

cc @lorilogan @aislin-ds